Gems/Atom/RHI/Code/Include/Atom/RHI/RayTracingPipelineState.h

```diff
@@ -24,6 +24,7 @@ namespace AZ
 
             AZ::Name m_rayGenerationShaderName;
             AZ::Name m_missShaderName;
+            AZ::Name m_callableShaderName;
```

Comment:
Arent hit/miss/etc callable shaders in principal. What is the advantage of callable shaders? Are you expecting to use them in specific features?

Reply:
It's a RT feature that allows the RayGen/ClosestHit/Miss shaders to invoke another shader that is scheduled separately by the GPU. This allows the main RT shaders to be simplified and invoke separate callable shaders to do different kinds of work, instead of putting it all in the main RT shaders and branching.